pkgs/tools/security/wpscan/default.nix

```diff
@@ -9,9 +9,7 @@ bundlerApp {
   gemdir = ./.;
   exes = [ "wpscan" ];
 
-  buildInputs = [
-    makeWrapper
-  ];
+  nativeBuildInputs = [ makeWrapper ];
```

Comment:
Please revert

Reply:
added support for nativeBuildInputs in [`3bfda6b` (#186564)](https://github.com/NixOS/nixpkgs/pull/186564/commits/3bfda6b18db43278f240d6598ba048c29375ec98)